ui2.0/app/reducers/password.js

```diff
@@ -0,0 +1,17 @@
+import { SET_REENTER } from 'constants/password';
+
+const initialState = {
+  showReenter: 0
```

Comment:
we may want use to `False` instead, I know you set it to 0 and 1 because you want to use it as the value of `opacity`, usually we don't do it this way, I commented a work around previously (also try not to assign a variable multiple meanings 😄)